fordel/src/main/java/no/nav/ung/fordel/kodeverdi/BrevkodeInformasjonUtleder.java

```diff
@@ -42,6 +42,7 @@ private static Map<String, BrevkodeInformasjon> createTitler() {
         brevkodeliste.add(new BrevkodeInformasjon(Brevkode.UNGDOMSYTELSE_SOKNAD.getOffisiellKode(), null, "Søknad om ungdomsytelse - UNG Søknad", FagsakYtelseType.UNGDOMSYTELSE, null, FordelBehandlingType.DIGITAL_SØKNAD, false));
         brevkodeliste.add(new BrevkodeInformasjon("UNG Endringssøknad", null, "Endringssøknad for ungdomsytelsen - UNG Endringssøknad", FagsakYtelseType.UNGDOMSYTELSE, null, FordelBehandlingType.DIGITAL_SØKNAD, false));
         brevkodeliste.add(new BrevkodeInformasjon(Brevkode.UNGDOMSYTELSE_INNTEKTRAPPORTERING.getOffisiellKode(), null, "Inntektsrapportering for ungdomsytelsen", FagsakYtelseType.UNGDOMSYTELSE, null, FordelBehandlingType.DIGITAL_SØKNAD, false));
+        brevkodeliste.add(new BrevkodeInformasjon(Brevkode.UNGDOMSYTELSE_OPPGAVE_BEKREFTELSE.getOffisiellKode(), null, "Bekreftelse av periodeendring for ungdomsprogrammet", FagsakYtelseType.UNGDOMSYTELSE, null, FordelBehandlingType.DIGITAL_SØKNAD, false));
```

Comment:
Burde denne også vært `UNG Oppgavebekreftelse`?

Reply:
Eg ser at dette kun er relevant for Gosysoppgaver, så vi kan mest sannsynlig fjerne tittel her

Reply:
Men eg tar dette i ein annen PR